LICENSE.md

```diff
@@ -8,7 +8,7 @@ Version 2.0.2
 
 **Game**: Minecraft, the Minecraft server, and all associated software or derivatives, as defined by Mojang.
 
-**Mod**: Software which modifies the Game, in any form (source, compiled binary, etc).
+**The Mod**: Any copyrightable work licensed under this license, in any form (source, compiled binary, etc).
```

Comment:
This license isn't intended for anything except for Minecraft mods. This definition is more general, but we don't want general. I'm definitely interested in a better definition of "Mod" though, what I have here isn't ideal.

Also, "The" is unnecessary.